factory/.env

```diff
@@ -8,10 +8,13 @@
 BASE_IMAGE='debian:bullseye-slim'
 
 # Node Versions: https://nodejs.org/en/download/releases/
-NODE_VERSION='18.14.1'
+FACTORY_DEFAULT_NODE_VERSION='18.15.0'
+
+# Node Versions: https://nodejs.org/en/download/releases/
+NODE_VERSION='18.15.0'
```

Comment:
What is the difference between the factory default node version and node version? Does the readme need this env listed explain when to use one or the other? and why bump the node version to trigger another factory bump?

Reply:
There are two default node version, one that is the default version for the factory and one that is the default version for the included/base/browsers image.

Previously those two node versions were tied together, with this change they can now be broken apart, this change has been made such that we could, if we chose. update the default version of included/base/browsers to a new major version without updating the default in the factory and thus triggering a major version of the factory.

The readme doesn't need to be updated `FACTORY_DEFAULT_NODE_VERSION` only relates to how we are building the factory itself, the `FACTORY_DEFAULT_NODE_VERSION` is not available in the factory as an arg.

The node version is updated here to keep inline with nodes lts version: https://nodejs.org/en

Reply:
updating the node version will also trigger a new image for each of the prebuilt images, which in turn will update the latest tag for all of them as well.

Reply:
> The readme doesn't need to be updated FACTORY_DEFAULT_NODE_VERSION only relates to how we are building the factory itself, the FACTORY_DEFAULT_NODE_VERSION is not available in the factory as an arg.

Sounds like we don't need to add the NODE_VERESION in the .env file then right? Cuz it would fallback & should always be in sync with the FACTORY_DEFAULT_NODE_VERSION for our builds

Reply:
Or when would these not match in our code base?

Reply:
For now we could set NODE_VERESION to FACTORY_DEFAULT_NODE_VERSION

If in the future we could decided to go to node 20 with the pre-built images but keep the factory itself on node 18 to avoid a major version release for the factory itself... it just gives us more options.

Reply:
i made this change